report_custom_filename/controllers/reports.py

```diff
@@ -17,10 +17,10 @@ def index(self, action, token):
         context = dict(http.request.context)
         context.update(action["context"])
         report_xml = http.request.session.model('ir.actions.report.xml')
```

Comment:
@pcs-sossia No need for another PR, you can just make the change and push to this branch in order to update this PR. This is definitely within scope of "API migration" IMO